ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -735,19 +774,19 @@ export const DataAccessRequirements = fp.flow(withProfileErrorModal)((spinnerPro
     }
   }, [code]);
 
-
+  // Active module: Required Modules: (rt steps + ducc) i.e the required steps user need to complete to grant RT Access
```

Comment:
could you add a little here to the effect of "an active module must be one of the required modules" or "only a required module can be active" ?

Reply:
I would still prefer a change.  How about:
```
// the activeModule must be one of the requiredModules
// the requiredModules represent those which are required for RT access: RT modules + DUCC
```

Reply:
Other way around :?
 // The requiredModules represent those which are required for RT access: RT modules + DUCC
  // The activeModule must be one of the requiredModules

Reply:
That works too!  Thanks.